### PR TITLE
Cloudinary

### PR DESCRIPTION
**Pull request by Mike-W00d**
Upload Functionality to Cloudinary working, and seeding of images into the MongoDB while working on automated transfer

### CODE REVIEW

**Comment by pablisch** [comment]
I had another look at the cloudinary PR, or at least the branch. It still does not work locally for me and can't reconcile why it works for Mike and not me so thought it worth a discussion rather than ploughing on. 
I fixed the font issue by adding "typeface-montserrat": "^1.1.13", to the dependencies which is in the current main.
Now I get console.js:213 No routes matched location "/upload-image" when navigating to the image upload which is correct. Again, I do not understand how this works for Mike so did not want to change. Changing navigation in Navbar to /upload fixes this for me. 
Then trying to upload an image gives me a log of 'Must supply api_key' which makes sense as this probably relates to Cloudinary.
 I would suggest updating the readme with any necessary info. Also, I have found that any hidden variables, e.g. the API key and the current DB password would be better as env variables, e.g. using dotenv, as it will make it much easier to deploy later.
Aside from the font and route above, it looks great.

**Comment by Mike-W00d** [comment]
Apologies I noticed the The upload-image in the navbar route after creating this.

**Comment by pablisch** [comment]
Ah, that makes sense. Could you add the font dependency and push the Navbar changes and we'll look again tomorrow. 👍🏼

**Comment by Mike-W00d** [comment]
Completed the above monserat not giving issues on latest push and Upload route changed on Navbar

File: backend/app.js [hunk 1]
```diff
@@ -1,53 +1,95 @@
...
 const userRoutes = require('./routes/user'); // import the router object, which is exported from user.js
 
-const app = express(); // call the express function, which returns an object with a listen method
+const app = express();
 
+const cors = require("cors");
+const { error } = require("console");
 // app.use(cors()); // call the use method, which adds a middleware function to the middleware stack, to deal with the CORS policy
+const bodyParser = require("body-parser");
+app.use(bodyParser.json({ limit: "10mb" }));
```
Comment (pablisch) [inline backend/app.js]: Is this doing the same thing as express.json later but with a different limit?

File: backend/app.js [hunk 2]
```diff
@@ -1,53 +1,95 @@
...
+    // call the catch method, which adds a callback function to the promise, to handle the failure case
+    console.log("Unable to connect to MongoDB Atlas!"); // log a message to the console
     console.error(error); // log the error to the console
   });
-  
-app.use(express.json()); // call the use method, which adds a middleware function to the middleware stack, to parse the request body
-// express.json() intercepts every request with json data content type, take the request  body and put it onto the request object as the body property.
-
 
+app.use(cors());
```
Comment (pablisch) [inline backend/app.js]: This seems to be repeating what the setHeaders is doing a little further down.

**Review by pablisch — APPROVED** [review]
Hi Mike, A couple of comments about the backend which you can do what you want with but otherwise approved

**Pull request merged**